integrations/discord/events.go

```diff
@@ -7,19 +7,51 @@ import (
 	"go.uber.org/zap"
 
 	"go.autokitteh.dev/autokitteh/integrations/discord/internal/vars"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
 func (h *handler) handleEvent(event any, eventType string) {
-	akEvent, err := h.transformEvent(event, eventType)
-	if err != nil {
+	var authorID string
+
+	// TODO(ENG-1546) - Add support for more event types
```

Comment:
No need to separate the declaration of `authorID` from its setting.
```suggestion
	// TODO(ENG-1546) - Add support for more event types.
	var authorID string
```